homeassistant/components/light/hyperion.py

```diff
@@ -28,7 +28,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.string,
+    vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): cv.ensure_list,
```

Comment:
I'm surprised that it was a string to begin with, as the default is a list of 3 values between 0–255.

Instead of using just `ensure_list`, you will need to validate it afterwards that all entries in the list are correct.  Since you always expect 3 values, ensure list is acutally not going to work, as that will always result in a list with 1 entry…

It seems you want to have something like `vol.All(list, vol.Length(min=3, max=3), [vol.Range(min=0, max=255)])`